includes/options/class-amp-options-manager.php

```diff
@@ -327,6 +328,63 @@ public static function update_analytics_options( $data ) {
 		return self::update_option( 'analytics', wp_unslash( $data ) );
 	}
 
+	/**
+	 * Renders the welcome notice on the 'AMP Settings' page.
+	 *
+	 * Uses the user meta values for the dismissed WP pointers.
+	 * So once the user dismisses this notice, it will never appear again.
+	 *
+	 * @todo: Add the full copy for this notice when it is decided.
+	 */
+	public static function render_welcome_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		$notice_id = 'amp-welcome-notice-1';
+		$dismissed = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+		if ( in_array( $notice_id, explode( ',', strval( $dismissed ) ), true ) ) {
+			return;
+		}
+
+		?>
+		<div class="notice notice-info is-dismissible" id="<?php echo esc_attr( $notice_id ); ?>">
+			<span class="amp-logo-icon"></span>
+			<h1><?php esc_html_e( 'Welcome to the AMP for WordPress plugin v1.0', 'amp' ); ?></h1>
+			<p><?php esc_html_e( 'Thank you for installing! Bring the speed and features of the open source AMP project to your site, the WordPress way, complete with the tools to support content authoring and website development.', 'amp' ); ?></p>
+			<h2><?php esc_html_e( 'What&#39;s New', 'amp' ); ?></h2>
+			<p><?php esc_html_e( 'From granular controls that help you create AMP content, to Core Gutenberg support, to a sanitizer that only shows visitors error-free pages, to a full error workflow for developers, this v1.0 release makes it easier than ever to bring a rich, performant experience to your WordPress site via AMP HTML.', 'amp' ); ?></p>
+		</div>
+
+		<script>
+		( function( $ ) {
+			$( function() {
+				// On dismissing the notice, make a POST request to store this notice with the dismissed WP pointers so it doesn't display again.
+				$( document ).on( 'click', <?php echo wp_json_encode( "#$notice_id .notice-dismiss" ); ?>, function() {
+					console.log( 'clicked' );
+					$.post( ajaxurl, {
+						pointer: <?php echo wp_json_encode( $notice_id ); ?>,
+						action: 'dismiss-wp-pointer'
+					} );
+				} );
+			} );
+		}( jQuery ) );
+		</script>
+
+		<style>
+			.amp-logo-icon {
+				background-image: url( "<?php echo esc_url( amp_get_asset_url( 'images/amp-logo-icon.svg' ) ); ?>" );
```

Comment:
HTML entities don't render properly inside of CSS. So best to just remove `esc_url()` I believe.